api/graphql/mutations/community.test.js

```diff
@@ -52,11 +52,26 @@ describe('moderation', () => {
   })
 
   describe('removeModerator', () => {
-    it('works', async () => {
+    it('just removes moderator role', async () => {
```

Comment:
For extra credit, you could add one that checks to ensure the mutation can't be used by a non-moderator :smile: